com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -219,7 +221,7 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
 
             // RenderTexture format depends on camera and pipeline (HDR, non HDR, etc)
             // Samples (MSAA) depend on camera and pipeline
-            m_CameraColorAttachment.Init("_CameraColorTexture");
+            m_ColorBufferSystem = new RenderTargetBufferSystem("_CameraColorAttachment");
```

Comment:
Do we set somehwere _CameraColorTexture as property? As users can depend on this property and now it will be gone.

Reply:
No, I will add it!